settings/acsf/post-install/post-install.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @file
+ * Factory Hook: post-install.
+ *
+ * This hook enables you to execute PHP after a new website is created
+ * in your subscription. Unlike most API-based hooks, this hook does not
+ * take arguments, but instead executes the PHP code it is provided.
```

Comment:
Add `@todo Remove this script after https://www.drupal.org/node/2752961 is resolved and the corresponding Drupal core version is set as a minimum requirement for BLT.`